packages/fluentui/react-northstar/src/components/Icon/SvgIcon.tsx

```diff
@@ -1,14 +1,11 @@
+import { getElementType, getUnhandledProps, useStyles, useTelemetry } from '@fluentui/react-bindings';
 import {
-  getElementType,
-  getUnhandledProps,
-  useStyles,
-  useTelemetry,
   SvgIconProps,
   svgIconClassName,
   svgIconDisplayName,
   svgIconHandledProps,
   SvgIconChildrenFn,
-} from '@fluentui/react-bindings';
+} from '@fluentui/react-icons-northstar';
```

Comment:
Why we need `SvgIcon` component at all? 🤔

Reply:
The only reason why I left the component is, if there are use cases where the svgs are dynamic, so that users can render them inside the utility component, without us requiring from them to create new components.